src/bar-chart.js

```diff
@@ -73,25 +98,36 @@ class BarChart extends AbstractChart {
       horizontalLabelRotation = 0,
       withInnerLines = true,
       showBarTops = true,
-      segments = 4
+      showTextTop = false,
```

Comment:
could you change the prop name to something more like `showValuesOnTopOfBars`, and add it to Readme and index.d.ts

Reply:
Please check latest commit. should fix what you ask for:)